airbyte-integrations/connectors/source-tmdb/source_tmdb/spec.yaml

```diff
@@ -0,0 +1,17 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Tmdb Spec
+  type: object
+  required:
+    - api_key
+    - movie_id
+  additionalProperties: true
+  properties:
```

Comment:
The API support language, are you planning to add this parameter to filter movies?

Reply:
Comment resolved: Added language in the stream path

Reply:
Added config